Documentation/gettingstarted/aws-eni.rst

```diff
@@ -0,0 +1,80 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _k8s_aws_eni:
+
+*********************************
+Setting up Cilium in AWS ENI mode
+*********************************
+
+Create an AWS cluster
+=====================
+
+Setup a Kubernetes on AWS. You can use any method you prefer, for the simplicity
```

Comment:
Nit: a Kubernetes --> "Kubernetes" or "a Kubernetes cluster".


", for" --> ", but for"